### DISCUSSION TITLE
Followed example code but can't bypass Cloudflare

### DISCUSSION THREAD

**Discussion by benllpSec** [seleniumbase/SeleniumBase#3201] [Q&A] [answered]
Used below example code but can't bypass Cloudflare CAPTCHA checkbox , anything I missed?

**from seleniumbase import SB
with SB(uc=True, test=True) as sb:
    url = "https://seleniumbase.io/apps/turnstile"
    sb.uc_open_with_reconnect(url, reconnect_time=2)
    sb.uc_gui_handle_captcha()
    sb.assert_element("img#captcha-success", timeout=3)
    sb.set_messenger_theme(location="top_left")
    sb.post_message("SeleniumBase wasn't detected", duration=3)**

My current setup:
OS: Windows 10 Pro H22
Chrome Version: 130.0.6723.59
SeleniumBase Version: 4.31.6
PyAutoGUI version: 0.9.54

**Comment by mdmintz** [comment, marked as answer]
This worked for me:

```python
from seleniumbase import SB

with SB(uc=True, test=True) as sb:
    url = "https://seleniumbase.io/apps/turnstile"
    sb.uc_open_with_reconnect(url)
    sb.uc_gui_click_captcha()
    sb.assert_element("img#captcha-success", timeout=3)
    sb.set_messenger_theme(location="top_left")
    sb.post_message("SeleniumBase wasn't detected", duration=3)
```

**Reply by benllpSec** [reply]
seems only uc_gui_click_captcha() and uc_gui_handle_captcha() not work in my environment. Is there any debug mode I can enable or log I can check?
